dom-manipulation/main.js

```diff
@@ -0,0 +1,24 @@
+let numClick = 0;
+const button = document.querySelector('.hot-button');
+const count = document.querySelector('.click-count');
+
+button.addEventListener('click', clickhandler);
+
+function clickhandler(event) {
+  numClick++;
+  count.textContent = numClick;
```

Comment:
This is nit-picky but if you notice, once you click on your button - the text goes from `Clicks: 0` to `1`, instead of `Clicks: 1`

Reply:
Thanks, I added the text "Clicks: " after clicking the button.